src/blob/file.rs

```diff
@@ -93,35 +99,42 @@ impl File {
     async fn write_at_sync(&self, offset: u64, buf: &[u8]) -> IOResult<()> {
         let buf = buf.to_vec();
```

Comment:
Do you plan to update write functions as a part of https://github.com/qoollo/pearl/pull/212?

Reply:
Yes, I'd like to change only read path here